.travis.yml

```diff
@@ -188,30 +188,36 @@ before_install:
               tfold ext.mongodb tpecl mongodb-1.6.0alpha1 mongodb.so $INI
           fi
 
+          echo extension = memcached.so >> $INI
```

Comment:
should be after the if I suppose, see failures

Reply:
wheps